api-docs/rst/dev-guide/api-operations/methods/post-pause-v1.0-tenantid-groups-groupid.rst

```diff
@@ -1,18 +1,25 @@
 
 
-.. _post-pause-group-policy-execution-v1.0-tenantid-groups-groupid-pause:
+.. _post-pause-v1.0-tenantid-groups-groupid:
 
-Pause group policy execution
+Pause group
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 .. code::
 
-    POST /v1.0/{tenantId}/groups/{groupId}/pause
+    POST /v1.0/{tenantId}/groups/groupId/pause
 
-This operation pauses the execution of all scaling policies for a specified scaling group.
-
-IMPORTANT: This method has not been implemented in the Auto Scale API yet and will be implemented in a future release.
+This operation pauses a specific scaling group. After a group is paused no
+executions on the group are allowed. This includes API-generated or scheduled
+policy executions. Triggering convergerence is also not allowed.
+Any existing convergence that is running will be stopped. Group configuration
+like min/max/cooldown and launch configuration changes like imageRef are allowed
+during a paused state. If a scaling group is paused, the group state will
+return ``"paused": true``. Use :ref:`resume <resume-group>` API operation to
+resume a paused group.
 
```

Comment:
This operation changes the state of the specified group to *paused*. When a group is paused, no policy or convergence operations are allowed. Any convergence operations in progress are stopped. Group configuration operations like min/max/cooldown and launch configurations like imageRef can run when a group is paused. You can resume a paused group by submitting a `resume <post-resume-v1.0-tenantid-groups-groupid>` request.